src/pages/HomePage.js

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+
+const Root = styled("div")(({ theme }) => ({
```

Comment:
why not just use <Box sx={} >

Reply:
What do you mean?

Reply:
https://mui.com/components/box/

I would use this component for some of the  css you are doing,  i would would not use the styled theme that you were doing in the component.